bgpd/bgp_fsm.c

```diff
@@ -935,9 +935,25 @@ static void bgp_update_delay_process_status_change(struct peer *peer)
    read/write and timer thread. */
 void bgp_fsm_change_status(struct peer *peer, int status)
 {
+	struct bgp *bgp;
 
 	bgp_dump_state(peer, peer->status, status);
 
+	bgp = peer->bgp;
+	if (status == Established)
+		bgp->active_peers++;
+	else if ((peer->status == Established) && (status != Established))
+		bgp->active_peers--;
+
+	if (BGP_DEBUG(neighbor_events, NEIGHBOR_EVENTS))
+		zlog_debug("%s : vrf %u, active_peers %d", __func__,
+				bgp->vrf_id, bgp->active_peers);
```

Comment:
a few nits:
* please fix the indentation.
* use `%u` to print unsigned numbers.
* s/active_peers/established peers/ as Nikos suggested.